vscode/extension/src/commands/tableDiff.ts

```diff
@@ -0,0 +1,556 @@
+import * as vscode from 'vscode'
+import { LSPClient } from '../lsp/lsp'
+import { isErr } from '@bus/result'
+import { CallbackEvent, RPCRequest } from '@bus/callbacks'
+import { getWorkspaceFolders } from '../utilities/common/vscodeapi'
+
+export function showTableDiff(
+  lspClient?: LSPClient,
+  extensionUri?: vscode.Uri,
+) {
+  return async () => {
+    if (!lspClient) {
+      vscode.window.showErrorMessage('LSP client not available')
+      return
+    }
+
+    if (!extensionUri) {
+      vscode.window.showErrorMessage('Extension URI not available')
+      return
+    }
+
+    // Get the current active editor
+    const activeEditor = vscode.window.activeTextEditor
+    let selectedModelInfo: any = null
+
+    if (!activeEditor) {
+      // No active editor, show a list of all models
+      const allModelsResult = await lspClient.call_custom_method(
+        'sqlmesh/all_models',
+        { textDocument: { uri: '' } },
```

Comment:
I am not certain we want to be using the `all_models` endpoint which was a legacy endpoint used for autocomplete.